Makefile

```diff
@@ -7,7 +7,9 @@ DOCKER_PROTO_BUILDER := docker run -v $(shell pwd):/workspace --workdir /workspa
 PROJECTNAME=$(shell basename "$(PWD)")
 HTTPS_GIT := https://github.com/celestiaorg/celestia-app.git
 PACKAGE_NAME          := github.com/celestiaorg/celestia-app/v3
-GOLANG_CROSS_VERSION  ?= v1.22.6
+# Before upgrading the GOLANG_CROSS_VERSION, please verify that a Docker image exists with the new tag.
+# See https://github.com/goreleaser/goreleaser-cross/pkgs/container/goreleaser-cross
+GOLANG_CROSS_VERSION  ?= v1.23
```

Comment:
```suggestion
GOLANG_CROSS_VERSION  ?= v1.23.1
```

Reply:
Thanks! When I made the PR, the Docker image for v1.23.1 didn't exist yet but now it does.